.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 Intel Corporation
+name: Pull Request
+
+on: [pull_request]
+
+concurrency:
+  group: ${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+  
+
+jobs:
+  build:
+    env:
+      DOCKER_REGISTRY: "docker.io/"
+      DOCKER_REPOSITORY: "badhrinathpa/"
+      VERSION: "PR-${{ github.event.pull_request.number }}"
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout and build
+      - uses: actions/checkout@v3
+
+      - name: Build Docker image
+        run: |
+          make docker-build
+      # Format the code
+      - name: Go Format
+        run: |
+          make fmt
+
+      - name: Show all CI changes
+        run: |
+          git --no-pager diff
+
+      # Build again and commit
```

Comment:
I see only build and no commit... Did I miss anything ?

Reply:
good catch. will add the commit part.